app/src/main/java/io/pslab/models/PSLabSensor.java

```diff
@@ -109,12 +83,31 @@
     public static final String ROBOTIC_ARM = "Robotic Arm";
     public static final String WAVE_GENERATOR = "Wave Generator";
     public static final String OSCILLOSCOPE = "Oscilloscope";
-
+    public static final String MULTIMETER = "Multimeter";
+    public final String KEY_LOG = "has_log";
+    public final String DATA_BLOCK = "data_block";
+    public boolean isRecording = false;
```

Comment:
Why all these changed? Previous state has them in a neat way. Revert

Reply:
Done :+1: